### ISSUE TITLE
bug: CLI Jest `-u` argument is not passed

### ISSUE THREAD

**Issue by duhem-s** [ionic-team/stencil#3828]
**Stencil version:**
```
 @stencil/core@2.13.0
```

**Current behavior:**
When I run the `test` script, I can not pass anymore the `-u` argument to update my snapshots.

**Expected behavior:**
Being able to update my snapshot using the CLI.

**GitHub Reproduction Link:**
https://github.com/duhem-s/stencil-cli

**Other information:**
Using the command `npm run test -u` with stencil@2.12.1 :
![image](https://user-images.githubusercontent.com/113420581/202398737-1545ad8a-7267-4659-825e-706a5cda273f.png)

With stencil@2.13.0 :
![image](https://user-images.githubusercontent.com/113420581/202399382-12f0acc3-c9b0-4159-9729-a26353957382.png)

**Comment by tanner-reits** [comment]
Hey @duhem-s 👋 

Thanks for reporting this issue and providing a reproduction! I hopped in and took a peek at the problem you encountered across a couple different versions of Stencil (2.12.1, 2.13.0, and 2.19.3 (latest)). Definitely some inconsistencies between the versions. So, I'll get this pulled into our backlog for us to take a look at!

In the meantime, if you're running Stencil 2.13.0, I was able to get the snapshots to update in the reproduction repo using the un-aliased Jest CLI flag:
`npm run test -- --updateSnapshot`

Hopefully that can keep you moving for now!

**Comment by rwaskiewicz** [comment]
@duhem-s Can you do us a favor and let us know what versions of node and npm you're using? From your project's directory, run:
```
npm --version
node --version
```
Thanks!

**Comment by duhem-s** [comment]
Hi @rwaskiewicz !

I'm using npm@6.14.17 and node@14.19.3

**Comment by alicewriteswrongs** [comment]
Hey @duhem-s, we now have a PR which I believe fixes this problem (#4124). I just published a dev build with those changes which could be installed like so:

```
npm install @stencil/core@3.1.0-dev.1678201586.203c481
```

after installing it in your reproduction case I'm able to do

```
npm test -- -u
```

and the snapshots update!

Would you mind trying that out to confirm that it works for you as well?

**Comment by rwaskiewicz** [comment]
The PR associated with this issue has landed and is included in today's [v3.2.0 release](https://github.com/ionic-team/stencil/releases/tag/v3.2.0). Should this issue reappear, please feel free to open a new issue. Thanks again!
